airflow/models/baseoperator.py

```diff
@@ -281,7 +277,7 @@ class derived from this one results in the creation of a task object,
     }
 
     # noinspection PyUnusedLocal
-    # pylint: disable=too-many-arguments,too-many-locals
+    # pylint: disable=too-many-arguments,too-many-locals, too-many-statements
```

Comment:
Looks good, just curious to know on what is causing`too-many-statements`

Reply:
Yeah. I was wondering too. Did not have time to look at it . Will do when doing final passes. Thanks for being curious.

Reply:
The base operator __init__ is just complex. I think splitting it into separate methods makes little sense so I think it is justified to put it there.